### PR TITLE
Tests bernoulli

### PR DESCRIPTION
**Pull request by A-Vzer**

### CODE REVIEW

**Review by bvdmitri — APPROVED** [review]

File: test/rules/bernoulli/test_marginals.jl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module RulesBernoulliMarginalsTest
+
+using Test
+using ReactiveMP
+using Random
+import ReactiveMP: @test_marginalrules
+
+@testset "marginalrules:Bernoulli" begin
+
+     @testset "out, p: (m_out::PointMass, m_p::Beta)" begin
```
Comment (bvdmitri) [inline test/rules/bernoulli/test_marginals.jl:10]: See comment for Beta

File: test/rules/bernoulli/test_out.jl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @testset "Belief Propagation: (m_p::PointMass)" begin
+
+        @test_rules [ with_float_conversions = true ] Bernoulli(:out, Marginalisation) [
+            (input = (m_p = PointMass(1.0),), output = Bernoulli(1.0)),
+            (input = (m_p = PointMass(0.2),), output = Bernoulli(0.2)),
+        ]
+
+    end
+
+    @testset "Belief Propagation: (q_p::PointMass)" begin
```
Comment (bvdmitri) [inline test/rules/bernoulli/test_out.jl:19]: See comments for Beta

**Comment by bvdmitri** [comment]
Approved, but requires minor changes and we can merge.

**Review by bartvanerp — APPROVED** [review]
Indeed, after fixing the naming of the test sets we can merge.

**Review by bvdmitri — APPROVED** [review]

**Pull request merged**